### ISSUE TITLE
🐛 firebase_admob: ^0.10.0+2 - no Ads iOS Offline then Online

### ISSUE THREAD

**Issue by MightyRai90** [FirebaseExtended/flutterfire#4030]
If you start Admob in Offline (Airplane) mode. Initialize Interstitial and Rewarded. Then switch to online. 
Nothing works... 

Really bad if a user has no internet at the start. And never receive ads until the next app (cold) start. 

thanks!

**Comment by darshankawar** [comment]
@MightyRai90 
Please provide `flutter doctor -v` and a minimal reproducible code sample that shows the issue.
Thanks.

**Comment by MightyRai90** [comment]
flutter doctor:

```
[✓] Flutter (Channel stable, 1.22.0, on Mac OS X 10.15.7 19H2, locale en-AT)
    • Flutter version 1.22.0 at /Users/raimundeissler/developement/flutter
    • Framework revision d408d302e2 (5 weeks ago), 2020-09-29 11:49:17 -0700
    • Engine revision 5babba6c4d
    • Dart version 2.10.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/raimundeissler/Library/Android/sdk
    • Platform android-29, build-tools 29.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.1, Build version 12A7403
    • CocoaPods version 1.8.4

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (1 available)
    • iPhone von Raimund (2) (mobile) • 00008101-000619EC0C91003A • ios • iOS 14.1

! Doctor found issues in 1 category.
```

the flutter and dart plugin are installed. A little bit buggy the whole thing ^^

```
// create
AdvertisingManager()  {
    FirebaseAdMob.instance.initialize(
      appId: "****",
    );
  }

// load
    RewardedVideoAd.instance.load(adUnitId: "****", targetingInfo: targetingInfo); 

// show
    RewardedVideoAd.instance.show();
```

I really think thats an issue from the firebase_admob package...

**Comment by MightyRai90** [comment]
seemed to be fixed in version 10.0.2 !

**Comment by darshankawar** [comment]
@MightyRai90 
Given your last comment above, I am going ahead and closing this. If you disagree, write in comments and I'll reopen.
Thanks.
